neo4j/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM neo4j:3.5.11
+FROM neo4j:3.5.11-enterprise
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-01T15:50:21Z" title="Tuesday, October 1st 2019, 5:50:21 pm +02:00">Oct 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,4 +1,4 @@
-FROM neo4j:3.5.11
+FROM neo4j:3.5.11-enterprise
```


But that would mean that people cannot run neo4j without license keys anymore? :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-01T16:15:39Z" title="Tuesday, October 1st 2019, 6:15:39 pm +02:00">Oct 1, 2019</time>_

---


well, apparently they don't check the keys, you just have to agree to the license
see https://github.com/Human-Connection/Human-Connection/pull/1787/files#diff-4e5e90c6228fd48698d074241c2ba760R60

but also, we, since we are deploying to production, are agreeing to the commercial license and must provide the license number if asked, whereas others who are using it to help develop the site, would be agreeing to the developer's license https://neo4j.com/licensing/

but yeah, I think if people were wanting to set it up in different places/contexts, they might want to change this to the non-enterprise version

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-01T21:12:24Z" title="Tuesday, October 1st 2019, 11:12:24 pm +02:00">Oct 1, 2019</time>_

---


Seriously? It just works out of the box and we don't have to setup our license keys or anything like that?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-01T22:02:12Z" title="Wednesday, October 2nd 2019, 12:02:12 am +02:00">Oct 2, 2019</time>_

---


it works with local docker dev... like if you run `docker-compose up` and go to localhost:7474 and log in, you can see in the bottom left that it says you are using `Enterprise` edition